apkid/rules/apk/packers.yara

```diff
@@ -218,7 +218,7 @@ rule dexprotector_b : packer
     $encrptcustom = /assets\/[A-Za-z0-9.]{2,50}\.mp3/
 
   condition:
-    is_apk and 2 of ($encrptlib_*) and $encrptcustom and
```

Comment:
Is there a sample which isn't caught by the old version of this rule?

This weakens the rule a bit, but it still looks OK. Is there anything else that could be added though?

Reply:
A sample that's not caught with this rule. Weakening the rule makes it to match perfectly.

Reply:
Is it possible to share the hash of the sample and add it as a comment?

Reply:
Done